### DISCUSSION TITLE
https://angular.love/a-gentle-introduction-into-change-detection-in-angular

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#356] [General]
# https://angular.love/a-gentle-introduction-into-change-detection-in-angular

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/a-gentle-introduction-into-change-detection-in-angular

**Comment by sebmat003** [comment]
Here’s a diagram that demonstrates what happens when Angular runs change detection for the parent component:


Diagram is missing
